src/router/index.js

```diff
@@ -51,7 +51,7 @@ const routes = [
     component: UnlockWallet
   },
   {
-    path: '/onboarding/setup/:passphrase?',
+    path: '/onboarding/setup/:passphrase?/:passphrase?',
```

Comment:
what does this mean? 😄  why passphrase twice?

Reply:
fixed, was a typo related and should be imported but we don't need it anymore